BDArmory/UI/BDArmorySetup.cs

```diff
@@ -2890,6 +2890,7 @@ void WindowSettings(int windowID)
                     BDArmorySettings.MISSILE_CM_SETTING_TOGGLE = GUI.Toggle(SLineRect(++line), BDArmorySettings.MISSILE_CM_SETTING_TOGGLE, StringUtils.Localize("#LOC_BDArmory_Settings_MissileCMToggle"));
                     if (BDArmorySettings.MISSILE_CM_SETTING_TOGGLE)
                     {
+                        BDArmorySettings.MISSILE_FUEL_DEDUCTION = GUI.Toggle(SLineRect(++line), BDArmorySettings.MISSILE_FUEL_DEDUCTION, StringUtils.Localize("#LOC_BDArmory_Settings_MissileFuelDeduction"));
```

Comment:
I've added an indent for these settings, so this should now use `SLineRect(++line, 1)`.